packages/core/src/alert/alert-group.stories.ts

```diff
@@ -137,50 +147,56 @@ export const alertGroup = () => {
 export const bannerGroupStatus = () => {
   return html`
     <div cds-layout="vertical gap:sm">
-      <cds-alert-group type="banner">
-        <cds-alert closable status="info">
+      <cds-alert-group type="banner" status="info">
+        <cds-alert closable>
           This example is a closable banner alert inside a banner alert group with a status of "info".
         </cds-alert>
       </cds-alert-group>
 
-      <cds-alert-group type="banner">
-        <cds-alert status="success" closable>
+      <cds-alert-group type="banner" status="success">
+        <cds-alert closable>
           This example is a closable alert with a status of "success" inside a banner alert group.
         </cds-alert>
       </cds-alert-group>
 
-      <cds-alert-group type="banner">
-        <cds-alert status="warning" closable>
+      <cds-alert-group type="banner" status="warning">
+        <cds-alert closable>
```

Comment:
Similar to angular data flow has to be top down with the prop setting since we cant control the render order. So the group can set the child elements but not the child. If we want the child to alert a parent state then we would need to use some additional custom events.